include/api.h

```diff
@@ -58,6 +58,7 @@
 #include "pros/optical.h"
 #include "pros/rtos.h"
 #include "pros/rotation.h"
+#include "pros/screen.h"
```

Comment:
Similar comment as before, given that our plans are to decouple LVGL from the Kernel in the near future, these are exposed by default intentionally so users can get used to this. Furthermore, given the hard to use nature and lack of officially hosted documentation many users would rather use the VEXCode style screen API instead of llemu or LVGL.

Reply:
Similar rationale as before in other comments.